src/rule.js

```diff
@@ -10,9 +10,11 @@
         email: /^([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/,
         number: /^\d+$/,
         minlength: function(value, scope, element, attrs, param) {
+          value += '';
```

Comment:
yes, this is valid.

But I would like to remove this commit out of this pr scope (using model value)

then we can have another pr/commit for separate `maxlength` and `maxvalue` but it doesn't really matter, it's just a rule and everyone can customise it. 🙂